knexfile.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: {
+      host: '127.0.0.1',
+      user: 'postgres',
+      password: process.env.PG_PASS,
```

Comment:
Nice work in pulling the password out of the sourced file - so many little details to remember!